test/index.html

```diff
@@ -33,6 +33,7 @@ <h1>JSONPath Tests</h1>
         import './test.errors.js';
         import './test.escaping.js';
         import './test.eval.js';
+        import './test.safe-eval.js';
```

Comment:
Not related to this line, but wanted to mention I'd like to preserve 100% code coverage and thereby avoid committing a badge with less than 100% coverage. If we really need to ignore some lines of code, we can do so with ignore statements.

Reply:
The code I added has 100% coverage already.

The only file which has no coverage is [bin/jsonpath-cli.js](https://github.com/JSONPath-Plus/JSONPath/blob/main/bin/jsonpath-cli.js).

Do you want me to handle that as well in this PR ?

Reply:
Ah, I see. It would be nice to restore 100%, sure, but I leave it up to you then.

Reply:
Added the tests for `bin/jsonpath-cli.js` as well.

I noticed that the cli is very minimal and lacks the options or good error reporting. But that should be out of scope for this PR.

---
PS:
I tried to be quick at deleting my inaccurate comment hoping no-one sees it, but I guess you also replied at the same moment.

For readers, the deleted message was like
> My code is already 100% test covered. The only file missing test coverage is [bin/jsonpath-cli.js](https://github.com/JSONPath-Plus/JSONPath/blob/324b8e52f7e12b321f07a625ebbde9340e692120/bin/jsonpath-cli.js) added in https://github.com/JSONPath-Plus/JSONPath/pull/206

I was inaccurate because the `ignoreEvalError` option I added at the last-minute had no tests and was breaking existing tests.